### PR TITLE
chore: readme cleanup post workflow change

### PR DESCRIPTION
**Pull request by jbadan**
Post #1345, there were a couple open tasks.

This pr 
1. cleans up a couple confusions - `npm run build:prod`, added `/scss` to package as we say we export it in the readme and should be consistent. 
1. cleans up a couple missed steps
1. removes additional readme files. These now live in the wiki pages: https://github.com/SAP/fundamental/wiki

I created a custom sidebar to help consumers/developers find what they are looking for. Feel free to change the order/topics/etc. It's a good starting point.

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit 674eb7f3fb4c51e680bf17664642c61219a51ec5

https://deploy-preview-1355--fundamental.netlify.com

File: README.md [hunk 1]
```diff
@@ -35,8 +36,19 @@ npm install fiori-fundamentals --save
...
+
+The distributed `less` folder is NOT a full port of the library to LESS. We are providing only select files that can be used in a LESS environment. For example, some `calc` functions are escaped to avoid errors when the library is processed as LESS.
+
+> NOTE: The `fonts` and `icons` are not included as part of this file. You will need to bring the regular `dist` versions of those files into your project separately.
+
+We are offering these files to ease the transition for some teams that are migrating their UI to a new look and feel but who are not in a position to adopt a SASS workflow.
+
+We welcome ideas and contributions to build this out if there is a strong need.
+
+
```
Comment (bcullman) [inline README.md]: is LESS being generated with every build?

Comment (jbadan) [reply README.md]: prod builds yes.

**Comment by bcullman** [comment]
@jbadan: There is still one post-merge item from this PR: #1345, to clean up the gh-pages branch right?

File: README.md [hunk 2]
```diff
@@ -35,8 +36,19 @@ npm install fiori-fundamentals --save
 
 > NOTE: We only distribute the [fonts, icons and CSS](https://github.com/SAP/fundamental/tree/master/dist) and [the SASS source](https://github.com/SAP/fundamental/tree/master/scss), not the full project or HTML for specific components.
 
+
+### LESS compatible Fiori Fundamentals
+
```
Comment (bcullman) [inline README.md]: This is way too much real estate to provide to LESS.
remove this text, add it to a wiki page, and link to it here.

Comment (bcullman) [reply README.md]: additionally, below this, perhaps have another heading here for icons, and a link to the icon wiki

**Comment by jbadan** [comment]
@bcullman yes, working with @eboyer later today on that.

**Pull request merged**